### PR TITLE
Support TargetPointMultiplier

### PR DESCRIPTION
**Pull request by isuruf**
WIP because it needs https://github.com/inducer/sumpy/pull/46

### CODE REVIEW

**Comment by isuruf** [comment]
macos build fails with

> ImportError: dlopen(/Users/runner/work/pytential/pytential/.miniforge3/envs/testing/lib/python3.9/site-packages/pyopencl/_cl.cpython-39-darwin.so, 2): Symbol not found: _clCloneKernel
  Referenced from: /Users/runner/work/pytential/pytential/.miniforge3/envs/testing/lib/python3.9/site-packages/pyopencl/_cl.cpython-39-darwin.so
  Expected in: flat namespace
  
cc @inducer

**Comment by isuruf** [comment]
A rerun fixed the pyopencl failure

**Comment by inducer** [comment]
Rebased to clean up the diff. Hope that's OK with you.

File: pytential/symbolic/primitives.py [hunk 1]
```diff
@@ -1692,9 +1692,10 @@ def int_g_vec(kernel, density, qbx_forced_limit, source=None, target=None,
     "vectorizing" behavior of of the constructor of :class:`IntG`
     for use cases where that is preferred.
     """
-    from sumpy.kernel import SourceDerivativeRemover, TargetDerivativeRemover
-    sdr = SourceDerivativeRemover()
-    tdr = TargetDerivativeRemover()
+    from sumpy.kernel import (SourceTransformationRemover,
+            TargetTransformationRemover)
+    sdr = SourceTransformationRemover()
+    tdr = TargetTransformationRemover()
```
Comment (inducer) [inline pytential/symbolic/primitives.py:1698]: Same renames as in sumpy.

**Review by inducer — COMMENTED** [review]
LGTM. Just one minor thing here.

**Pull request merged**